documentation/graphing-overview/index.html.md

```diff
@@ -0,0 +1,285 @@
+	});
+
+We use the `operations.publish()` to guarantee that any subscribers are notified of the call.
+
+Finally, use the Flow.js [Operations Channel](../generated/channels/operations-channel/) to subscribe to the operation you've just called. The second argument is a callback function, called whenever the model operation is called. We want to update and re-render our visualization each time this operation happens:
+
+	Flow.channel.operations.subscribe('updateXY',
+		function(data) {
+			var composedData = {
+				x: data.updateXY.arguments[0],
+				y: data.updateXY.arguments[1]
+			};
+			myThirdChartData.push(composedData);
+			myThirdChart.setData(myThirdChartData);
+			myThirdChart.render();
+	});
+
+The argument passed to the callback (`data`) is an object with the names and values of the arguments used to call the operation. 
+
+In this example, we're assuming that these arguments are scalars whose values are input directly by the user. The variable `composedData` uses these values to create a new (x,y) data point. Then, we need to explicitly add (`push`) this new data point onto the data set we're graphing.
+
+<a name="example"></a>
+**Putting it all together.**
+
+Here's the complete sample code:
+
+	<html>
+
+	<head>
+
+		<!-- for Contour -->
+		<link rel="stylesheet" href="http://forio.com/tools/contour/contour.min.css">
+		<script src="http://d3js.org/d3.v3.js"></script>
+		<script src="http://cdnjs.cloudflare.com/ajax/libs/lodash.js/2.4.1/lodash.js"></script>
+		<script src="https://forio.com/tools/contour/contour.js"></script>
+
+		<!-- for Flow.js -->
+		<script src="//ajax.googleapis.com/ajax/libs/jquery/2.0.2/jquery.min.js"></script>
+		<script src="https://forio.com/tools/js-libs/1.5.0/epicenter.min.js"></script>
+		<script src="https://forio.com/tools/js-libs/flow/0.9.0/flow.js"></script>
+
+	</head>
+
+	<body data-f-model="supply-chain-game.py">
+
+		<p>Here is a div with the Contour chart, used in Steps 1-4: </p>
+		<div class="myChart"></div>
+		<p>The next data point is added to the chart each time the user enters a new value:</p>
+		<input data-f-bind="sampleInt"></input>
+		<hr>
+
+		<p>Here is a div with the Contour chart used in Step #5, based on two variables updated automatically when the model is advanced: </p>
+		<div class="mySecondChart"></div>
+		<p><button data-f-on-click='advanceModel'>Advance Model</button></p>
+		<hr>
+		
+		<p>Here is a div with the Contour chart used in Step #6, based on two variables whose values are entered by the user: </p>
+		<div class="myThirdChart"></div>
+		<p>
+			Enter x-coordinate: <input id="x" type="text"></input><br>
+			Enter y-coordinate: <input id="y" type="text"></input></br>
+			<button id="submitButton">Update x,y</button>
+		</p>
+		
+		<script>
+
+			Flow.initialize();
+
+			// used in Step #6
+			$("#submitButton").click(function(){
+				Flow.channel.operations.publish('updateXY', [$("#x").val(), $("#y").val()]);
+			});	
+
+			$(function () {
```

Comment:
You can remove the `$(function () {`, you don't need it if you code is at the bottom of the page